docs/releasenotes/index.rst

```diff
@@ -0,0 +1,23 @@
+Release notes
+=============
+
+v3.0.0rc2
+------
+
+The development team is very happy to announce this release which focused on modernization and security fixes, as well as the dockerization, environment setup, and documentation.
+
+The main purpose of this release is to address security issues with how user's work is run on remote worker nodes.
```

Comment:
There really weren't any security fixes in this dev-cycle. The main purpose was modernization.